posthog/temporal/data_imports/sources/generated_configs.py

```diff
@@ -0,0 +1,223 @@
+
+
+@config.config
+class VitallyRegionConfig(config.Config):
+    subdomain: str
+    selection: Literal["EU", "US"] = "EU"
+
+
+@config.config
+class BigQuerySourceConfig(config.Config):
+    key_file: BigQueryKeyFileConfig
+    dataset_id: str
+    temporary_dataset: BigQueryTemporaryDatasetConfig | None = config.value(
+        alias="temporary-dataset", default_factory=lambda: None
+    )
+    dataset_project: BigQueryDatasetProjectConfig | None = None
+
+
+@config.config
+class BrazeSourceConfig(config.Config):
+    pass
+
+
+@config.config
+class ChargebeeSourceConfig(config.Config):
+    api_key: str
+    site_name: str
+
+
+@config.config
+class DoItSourceConfig(config.Config):
+    api_key: str
+
+
+@config.config
+class GoogleAdsSourceConfig(config.Config):
+    customer_id: str
+    google_ads_integration_id: int = config.value(converter=config.str_to_int)
+
+
+@config.config
+class GoogleSheetsSourceConfig(config.Config):
+    spreadsheet_url: str
+
+
+@config.config
+class HubspotSourceConfig(config.Config):
+    hubspot_integration_id: int = config.value(converter=config.str_to_int)
+
+
+@config.config
+class KlaviyoSourceConfig(config.Config):
+    pass
+
+
+@config.config
+class MSSQLSourceConfig(config.Config):
+    host: str
+    database: str
+    user: str
+    password: str
+    schema: str
+    port: int = config.value(converter=int)
+    ssh_tunnel: SSHTunnelConfig | None = None
+
+
+@config.config
+class MailchimpSourceConfig(config.Config):
+    pass
+
+
+@config.config
+class MailjetSourceConfig(config.Config):
+    pass
+
+
+@config.config
+class MetaAdsSourceConfig(config.Config):
+    account_id: str
+    meta_ads_integration_id: int = config.value(converter=config.str_to_int)
+
+
+@config.config
+class MongoDBSourceConfig(config.Config):
+    connection_string: str
+
+
+@config.config
+class MySQLSourceConfig(config.Config):
+    host: str
+    database: str
+    user: str
+    password: str
+    schema: str
+    port: int = config.value(converter=int)
+    using_ssl: bool = config.value(default=config.str_to_bool("true"), converter=config.str_to_bool)
+    ssh_tunnel: SSHTunnelConfig | None = None
+
+
+@config.config
+class PostgresSourceConfig(config.Config):
+    host: str
+    database: str
+    user: str
+    password: str
+    schema: str
+    port: int = config.value(converter=int)
+    connection_string: str | None = None
+    ssh_tunnel: SSHTunnelConfig | None = None
+
+
+@config.config
+class RedshiftSourceConfig(config.Config):
+    pass
+
+
+@config.config
+class SalesforceSourceConfig(config.Config):
+    salesforce_integration_id: int = config.value(converter=config.str_to_int)
+
+
+@config.config
+class SnowflakeSourceConfig(config.Config):
+    account_id: str
+    database: str
+    warehouse: str
+    auth_type: SnowflakeAuthTypeConfig
+    schema: str
+    role: str | None = None
+
+
+@config.config
+class StripeSourceConfig(config.Config):
+    stripe_secret_key: str
```

Comment:
Why the `stripe_` prefix? Not needed here. It's like having a table `foo` and naming the PK `foo_id` rather than jist `id`

Reply:
Easier to rename now that we've done a lot of work to migrate to the new structure than later :)

Will let you pick your fights tho, big PR indeed :)